tests/circuit_graph/test_qasm.py

```diff
@@ -227,12 +227,12 @@ def test_basis_state_initialization_decomposition(self):
     def test_unsupported_gate(self):
         """Test an exception is raised if an unsupported operation is
         applied."""
-        U = np.array([[1, 1], [1, -1]]) / np.sqrt(2)
-
         with qml.tape.QuantumTape() as circuit:
-            qml.S(wires=0), qml.QubitUnitary(U, wires=[0, 1])
+            qml.S(wires=0), qml.DoubleExcitationPlus(0.5, wires=[0, 1, 2, 3])

```

Comment:
It was actually hard to find an operation that didn't have a decomposition into QASM gates that I could use here... but this is a good thing :grin: